src/models/model.py

```diff
@@ -4,15 +4,15 @@
 import torch
 import torch.nn.functional as F
 
-from src.models.unet import UNet
-from src.data.data import DaVinciDataModule
+from models.unet import UNet
+from metrics.fid import calculate_fid
+from data.data import DaVinciDataModule
```

Comment:
Action required: `export PYTHONPATH="${PYTHONPATH}:/home/js11133/capstone/da_vinci_depth/src"` to ~/.bash_profile